src/core/scene/a-scene.js

```diff
@@ -312,6 +343,11 @@ module.exports = registerElement('a-scene', {
         renderer.setPixelRatio(window.devicePixelRatio);
         renderer.sortObjects = false;
         this.effect = new THREE.VREffect(renderer);
+        this.effect.autoSubmitFrame = false;
+
+        // Create a render target for post processing.
+        this.renderTarget = new THREE.WebGLRenderTarget(1, 1, { minFilter: THREE.LinearFilter, magFilter: THREE.NearestFilter, format: THREE.RGBAFormat });
+        this.renderTarget.texture.generateMipmaps = false;
```

Comment:
yeah it could even though the framebuffer is created only on the scene element so probably only once in the whole app.